node/src/checkpointers.test.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2021 IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as checkpointers from './checkpointers';
+import path from 'path';
+import fs from 'fs';
+import { CheckpointerState, FileCheckPointer } from './filecheckpointer';
+import { InmemoryCheckPointer } from './inmemorycheckpointer';
+
+async function writeToFile(
+    checkpointerstate: CheckpointerState,
+    path: string
+): Promise<void> {
+    const data = Buffer.from(JSON.stringify(checkpointerstate));
+    await fs.promises.writeFile(path, data);
+}
+
+const dir = './testUtils';
+const fileName = 'checkpoint.json';
+
+const checkPointerFilePath = path.join(dir, fileName);
```

Comment:
Tests should not be writing new files to the source/build directory. Use (and maybe later remove) a temp file instead. There is example of how to do this here: https://github.com/hyperledger/fabric-sdk-node/blob/5741f6eaf78b055e49fc42bcf95366d85222a199/fabric-network/test/testutils.ts#L61-L64